static/skywire-manager-src/src/app/pipes/auto-scale.pipe.ts

```diff
@@ -1,20 +1,76 @@
 import { Pipe, PipeTransform } from '@angular/core';
 import { BigNumber } from 'bignumber.js';
 
-export class AutoScalePipeParams {
+/**
+ * Params AutoScalePipe can receive.
+ */
+class AutoScalePipeParams {
+  /**
+   * If true, the numeric value will be shown.
+   */
   showValue: boolean;
+  /**
+   * If true, the unit will be shown.
+   */
   showUnit: boolean;
+  /**
+   * If true, the unit will be in data per second (like KB/s instead of KB).
+   */
+  showPerSecond: boolean;
+  /**
+   * If true, the numeric value will have at most 1 decimal.
+   */
+  limitDecimals: boolean;
+  /**
+   * If the data must be shown in bits (true) or bytes (false).
+   */
+  useBits: boolean;
 }
 
+/**
+ * Allows to convert a bytes value to KB, MB, GB, etc. It considers 1024, and not 1000, a K.
+ */
 @Pipe({
   name: 'autoScale'
 })
 export class AutoScalePipe implements PipeTransform {
+  private static readonly accumulatedMeasurements = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+  private static readonly measurementsPerSec = ['B/s', 'KB/s', 'MB/s', 'GB/s', 'TB/s', 'PB/s', 'EB/s', 'ZB/s', 'YB/s'];
+  private static readonly accumulatedMeasurementsInBits = ['b', 'Kb', 'Mb', 'Gb', 'Tb', 'Pb', 'Eb', 'Zb', 'Yb'];
+  private static readonly measurementsPerSecInBits = ['b/s', 'Kb/s', 'Mb/s', 'Gb/s', 'Tb/s', 'Pb/s', 'Eb/s', 'Zb/s', 'Yb/s'];
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
  private static readonly measurementsPerSecInBits = ["b/s", 'Kb/s', 'Mb/s', 'Gb/s', 'Tb/s', 'Pb/s', 'Eb/s', 'Zb/s', 'Yb/s'];
```